image/image.go

```diff
@@ -120,7 +120,7 @@ func unpack(w walker, dest, refName string) error {
 		return err
 	}
 
-	if err = ref.validate(w, validRefMediaTypes); err != nil {
+	if err = ref.validate(w, []string{v1.MediaTypeImageManifest}); err != nil {
```

Comment:
> manifest list points to only a few special manifests, just specify its platform, there is no other property 

I don't know if you understand the `manifest list` correctly :).  How could you unpack a OCI image if the `ref`( for more information about ref https://github.com/opencontainers/image-spec/blob/master/image-layout.md) points to  manifest list ? and how do you know the `ref` is always point to a manifest ?